### ISSUE TITLE
QUESTION: Is GenomeFasta reverse complementing negative strand sequences on write?

### ISSUE THREAD

**Issue by adamklie** [ML4GLand/SeqData#13]
I noticed that for the following lines, it seems we are automatically reverse complementing reverse (-) strand sequences, on write to Zarr:

https://github.com/ML4GLand/SeqData/blob/main/seqdata/_io/readers/fasta.py#L220
https://github.com/ML4GLand/SeqData/blob/main/seqdata/_io/readers/fasta.py#L249

I wanted to better understand the logic here. If I pass in a bed file with strand information, are we effectively writing everything as if it were on the forward (+) strand?

I think this may be a problem if so. If I read back in this same dataset, I'm going to get the (+) strand returned, but the metadata for the dataset is going to indicate that it is coming from the original strand, which could be (-).

**Comment by adamklie** [comment]
Also seems like we are assuming positive strand if that column isn't provided. I think this is fine. We should just have it documented clearly:

https://github.com/ML4GLand/SeqData/blob/main/seqdata/xarray/seqdata.py#L345

**Comment by d-laub** [comment]
Yes, the intent is that negative stranded sequences should be RC'd and values should be reversed. Agree that this should be documented clearly.
